sdk/python/feast/infra/offline_stores/contrib/spark_offline_store/spark.py

```diff
@@ -449,7 +449,13 @@ def _get_entity_df_event_timestamp_range(
         # If the entity_df is a string (SQL query), determine range
         # from table
         df = spark_session.sql(entity_df).select(entity_df_event_timestamp_col)
+
+        # Checks if resulting data frame has any data, if not
+        # raises a meaningful error
+        spark_utils._assert_query_success(df, entity_df)
```

Comment:
Not sure if these these utility methods should be in a different file - they're pretty small. Can't they just be defined in this same file?

Reply:
> Not sure if these these utility methods should be in a different file - they're pretty small. Can't they just be defined in this same file?

Just following the existing project structure for this. Didn't want to overload `spark.py` ; may be, as feast-spark matures from contrib, this structure may be useful  ?

Reply:
No i think we don't need a new file just for this, we can add it later if the need to organize code differently arises